onnxscript/ir/_linked_list.py

```diff
@@ -1,24 +1,16 @@
 """Mutable list for nodes in a graph with safe mutation properties."""
```

Comment:
Should add usual copyright notice to new files:
```
# -------------------------------------------------------------------------
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.
# --------------------------------------------------------------------------
```

Reply:
Done